HeterogeneousCore/SonicTriton/src/RetryActionDiffServer.cc

```diff
@@ -0,0 +1,36 @@
+#include "HeterogeneousCore/SonicTriton/interface/RetryActionDiffServer.h"
+#include "HeterogeneousCore/SonicTriton/interface/TritonClient.h"
+#include "HeterogeneousCore/SonicTriton/interface/TritonService.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+
+RetryActionDiffServer::RetryActionDiffServer(
+  const edm::ParameterSet& conf, 
+  SonicClientBase* client
+): RetryActionBase(conf, client) {}
+
+void RetryActionDiffServer::start() {
+  this->shouldRetry_ = true;
+}
+
+void RetryActionDiffServer::retry() {
+  if (!this->shouldRetry_) {
+    this->shouldRetry_ = false;
+    edm::LogInfo("RetryActionDiffServer") << "Retry not armed; skipping.";
+    return;
+  }
+
+  try {
+    auto* tritonClient = static_cast<TritonClient*>(client_);
+    edm::LogInfo("RetryActionDiffServer") << "Attempting retry by switching to fallback server";
+    tritonClient->updateServer(TritonService::Server::fallbackName);
+    eval();
+  } catch (const std::exception& e) {
```

Comment:
This should follow the existing pattern:
```
  catch (TritonException& e) {
    e.convertToWarning();
```
(see https://github.com/cms-sw/cmssw/blob/79b590e4ef3ac7018721083e171b90f1d6b13e8f/HeterogeneousCore/SonicTriton/src/TritonClient.cc#L335, https://github.com/cms-sw/cmssw/blob/79b590e4ef3ac7018721083e171b90f1d6b13e8f/HeterogeneousCore/SonicTriton/src/TritonMemResource.cc#L30)